datastore/datastore.go

```diff
@@ -91,6 +147,26 @@ func init() {
 	boltdb.Register()
 }
 
+// DefaultScopes returns a map of default scopes and it's config for clients to use.
+func DefaultScopes(dataDir string) map[string]*ScopeCfg {
```

Comment:
Sure. but still, -g option is a one time initialization parameter and hence I prefer an explicit function to set the base prefix than overloading this get function.